src/composables/verifySignedIn.js

```diff
@@ -0,0 +1,14 @@
+// A function to verify if user is singed in or not
+
+function SignedIn(){
+
+//Fetches the current user from firebase 
+
+	const user = firebase.auth().currentUser;
```

Comment:
Please make use of the configuration auth state not a newly initialized one.

There's also no imports to your code